### PR TITLE
Add a version to web3torrent

### PR DESCRIPTION
**Pull request by lalexgap**
Uses [git-revision-webpack-plugin](https://www.npmjs.com/package/git-revision-webpack-plugin) to add a version to the footer.

### CODE REVIEW

File: packages/web3torrent/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "html-webpack-plugin": "4.0.0-beta.5",
     "identity-obj-proxy": "3.0.0",
     "is-wsl": "^1.1.0",
+    "git-revision-webpack-plugin": "3.0.6",
```
Comment (geoknee) [inline packages/web3torrent/package.json:38]: Should probably be a `devDependency` ?

File: packages/web3torrent/config/webpack.config.js [hunk 2]
```diff
@@ -547,7 +550,12 @@ module.exports = function(webpackEnv) {
       // It is absolutely essential that NODE_ENV is set to production
       // during a production build.
       // Otherwise React will be compiled in the very slow development mode.
-      new webpack.DefinePlugin(stringifiedEnv),
+      new webpack.DefinePlugin({
+        ...stringifiedEnv,
+        VERSION: JSON.stringify(gitRevisionPlugin.version()),
+        COMMIT_HASH: JSON.stringify(gitRevisionPlugin.commithash()),
+        BRANCH: JSON.stringify(gitRevisionPlugin.branch())
+      }),
```
Comment (geoknee) [inline packages/web3torrent/config/webpack.config.js:558]: When I tried this out, I did not see these new variables from the output of `console.log(process.env)`
Screenshot 2020-05-07 at 10 13 18

File: packages/web3torrent/src/constants.ts [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import {Status, TorrentUI} from './types';
 import {ChannelState} from './clients/payment-channel-client';
 import {utils} from 'ethers';
+console.log(process.env);
```
Comment (geoknee) [inline packages/web3torrent/src/constants.ts:4]: ✂️ 
Screenshot 2020-05-07 at 10 21 18

File: packages/web3torrent/config/webpack.config.js (hunk 2 above)
Comment (lalexgap) [reply packages/web3torrent/config/webpack.config.js]: Oops, I did a lot of rebases and broke this. Should be fixed now.

Comment (geoknee) [reply packages/web3torrent/config/webpack.config.js]: Hmm I still have this problem when checking out this branch. Does it work for you?

Comment (geoknee) [reply packages/web3torrent/config/webpack.config.js]: Hang on I didn't pull your latest changes...

**Pull request merged**